src/uwtools/config/tools.py

```diff
@@ -38,6 +37,43 @@ def compare_configs(
     return cfg_a.compare_config(cfg_b.data)
 
 
+def config_check_depths_dump(config_obj: Union[Config, dict], target_format: str) -> None:
+    """
+    Check that the depth does not exceed the target format's max.
+
+    :param config_obj: The reference config dictionary.
+    :param target_format: The target format.
+    :raises: UWConfigError on excessive config object dictionary depth.
+    """
+    # define a function with the conditions of an invalid depth
```

Comment:
nitpick: Capitalize comment sentence and end with a period, here and below.

Reply:
Aaaah! Thanks lol